Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM golang:alpine
+
+EXPOSE 6667/tcp 6697/tcp 8080/tcp
+
+RUN \
+    apk add --update git && \
+    rm -rf /var/cache/apk/*
+
+RUN mkdir -p /go/src/github.com/oragono/oragono
+WORKDIR /go/src/github.com/oragono/oragono
+
+COPY . /go/src/github.com/oragono/oragono
+
+RUN go get -v -d
```

Comment:
this ends up getting a different version of dependencies than pinned in `vendor` via submodules I believe.

Since we grabbed git anyways, it seems sane to use that to initialize submodules and not deal with `go get` at all.